pypi_formula_mappings.json

```
@@ -797,8 +797,8 @@
     "exclude_packages": ["certifi"]
   },
   "virt-manager": {
-    "exclude_packages": ["certifi"],
-    "extra_packages": ["libvirt-python", "requests"]
+    "exclude_packages": ["certifi", "libvirt-python"],
```

before it needs extra-package to include the resource, now you just need to remove it from extra, which would be good enough?

```suggestion
    "exclude_packages": ["certifi"],
```

---

Okay